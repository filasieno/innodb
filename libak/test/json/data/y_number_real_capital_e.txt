name=y_number_real_capital_e
max_depth=32
max_string_size=2048
max_json_size=1048576
----------
[1E22]