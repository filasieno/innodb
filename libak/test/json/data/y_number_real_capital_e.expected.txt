---
STATE_CHANGED_EVENT: STATE_INITIALIZED
BEGIN_ARRAY
FLOAT 1e+22
END_ARRAY
STATE_CHANGED_EVENT: STATE_DONE
